test/mlir/accelerators/nnpa/conversion/onnx-to-zhigh/pool.mlir

```diff
@@ -101,6 +101,19 @@ func @averagepool_should_lower_to_zhigh_same_upper(%arg0: tensor<1x3x32x32xf32>)
 }
 
 // -----
+// CHECK-LABEL:  func @test_pool_not_lowered_1d
```

Comment:
Should the function name be changed to reflect what it is supposed to check?

Reply:
I used this function name since there is a test for 3d (`test_pool_not_lowered_3d`), but not for 1d. And if there is this test for 1d, I thought we could have avoided this bug. 
You mean we should use more specific name for this PR such as `test_pass_pool_1d_dilation` or something?

Reply:
`test_pass_pool_1d_dilation` would be better, but again the ONNX op here was not lowered to ZHigh was not because of dilation I think. To test your code about dilation, you could have a ONNX op that satisfies other conditions but not dilation.

Anyway, `test_pool_not_lowered_3d` is not good too. I don't what `3d` is for. If you know it, please correct it too. Thanks!

Reply:
`1d` and `3d` should come from maxpool1d and maxpool3d which are not supported in zDNN. So, I think they are not lowered because input shape is not 4d.

Reply:
@tungld Update the name. Thanks!

Reply:
Also simplify the function.